core/src/components/datetime/datetime.tsx

```diff
@@ -912,6 +885,29 @@ export class Datetime implements ComponentInterface {
         const monthBox = month.getBoundingClientRect();
         if (Math.abs(monthBox.x - box.x) > 2) return;
 
+        /**
+         * If we're force-rendering a month, and we've scrolled to
+         * that month, return that.
+         *
+         * Checking that we've actually scrolled to the forced month
+         * is mostly for safety; in theory, if there's a forced month,
+         * that means a new value was manually set, so we should have
+         * automatically animated directly to it.
+         */
+        const { forceRenderDate } = this;
+        const firstDayEl = month.querySelector('.calendar-day');
+        const dataMonth = firstDayEl?.getAttribute('data-month');
+        const dataYear = firstDayEl?.getAttribute('data-year');
+        if (
+          forceRenderDate !== null &&
```

Comment:
Is there a scenario where we could return the forced month even if it hasn't scrolled into view? Wondering if we'd be fine to just return the forced date if `forceRenderDate !== null`

Reply:
As mentioned in the comments above this, the extra check is out of an abundance of caution. I was nervous that we might see bugs if `forceRenderDate` is ever set outside the specific circumstance of scrolling to it immediately. I can remove the extra checks if you don't feel this is useful at the moment, though.

Reply:
I think it's worth removing for now. `forceRenderDate` (as it stands right now) should only be used as a tool to scroll to the new month when updating the `value` async. If that remains true, then we can assume that month/year has changed.

Might be worth adding a comment to `forceRenderDate` surrounding its intended usage if one does not already exist.